src/makeInfinityAdder.js

```diff
@@ -6,7 +6,24 @@
  */
 
 function makeInfinityAdder() {
-  // write code here
+  return (firstNumber) => {
+    let sum = firstNumber;
+
+    if (firstNumber === undefined) {
+      sum = 0;
+
+      return sum;
+    }
```

Comment:
So here you check if the number is undefined, and you do the same check below in add function. I think it's better to use one function to return and check input values once

Reply:
Thanks, remaked